libvirt/README.md

```diff
@@ -58,3 +58,224 @@ mkdir -p /opt/data-dir
+$ kubectl get nodes
+NAME                 STATUS   ROLES                  AGE     VERSION
+peer-pods-master-0   Ready    control-plane,master   6m8s    v1.25.3
+peer-pods-worker-0   Ready    worker                 2m47s   v1.25.3
+```
+
+## Prepare the Pod VM volume
+
+In order to build the Pod VM without installing the build tools, you should use the `docker-build` Makefile target that will
+run the entire process inside a container.
+
+Ensure that the Kata Containers repository is cloned side-by-side with the cloud-api-provider's, as both repositories will be
+mounted as volumes in the container environment. Refer to [docs/DEVELOPMENT.md](/docs/DEVELOPMENT.md) for further
+information about setup of the sources.
+
+Then run:
+
+```
+$ cd image
+$ make docker-build
+```
+
+The qcow2 image file will be created at the `output` directory as shown below:
+
+```
+$ ls output/
+podvm-7da706d-dirty-amd64.qcow2
+```
+
+Next you will need to create a volume on libvirt's system storage and upload the image content. That volume is used by
+the cloud-api-adaptor program to instantiate a new Pod VM. Still on the `image` directory, you should run `make push` just
+like below:
+
+```
+$ make push
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  5394  100  5394    0     0   6421      0 --:--:-- --:--:-- --:--:--  6421
+virsh -c qemu:///system vol-create-as --pool default --name podvm-base.qcow2 \
+	--capacity 107374182400 --allocation 2361393152 --prealloc-metadata \
+	--format qcow2
+Vol podvm-base.qcow2 created
+
+virsh -c qemu:///system vol-upload --vol podvm-base.qcow2 output/podvm-7da706d-dirty-amd64.qcow2 \
+	--pool default --sparse
+```
+
+You should see that the `podvm-base.qcow2` volume was proper created:
+
+```
+$ virsh -c qemu:///system vol-info --pool default podvm-base.qcow2
+Name:           podvm-base.qcow2
+Type:           file
+Capacity:       6.00 GiB
+Allocation:     631.52 MiB
+```
+
+## Install and configure Confidential Containers and cloud-api-adaptor in the cluster
+
+The easiest way to install the cloud-api-adaptor along with Confidential Containers in the cluster is through the
+Kubernetes operator available in the `install` directory of this repository.
+
+Start by creating a public/private RSA key pair that will be used by the cloud-api-provider program, running on the
+cluster workers, to connect with your local libvirtd instance without password authentication. Assume you are in the
+`libvirt` directory, do:
+
+```
+$ cd ../install/overlays/libvirt
+$ ssh-keygen -f ./id_rsa -N ""
+$ cat id_rsa.pub >> ~/.ssh/authorized_keys
+```
+**Note**: ensure that `~/.ssh/authorized_keys` has the right permissions (read/write for the user only) otherwise the
+authentication can silently fail.
+
+You will need to figure out the IP address of your local host (e.g. 192.168.1.107). Then try to remote connect with
```

Comment:
isn't it defaults to 192.168.122.1 ? maybe because you non-root?

Reply:
default is usually 192.168.122.1. So may be the example can be changed to that.

Reply:
But it's fine as-is as well.

Reply:
Done.